arm-network/2017-06-01/swagger/networkInterface.json

```diff
@@ -587,6 +598,20 @@
           "type": "string",
           "description": "The destination port or range."
         },
+        "sourcePortRanges": {
```

Comment:
@DeepakRajendranMsft Can you please respond here? are these settable properties?